tablepyxl/style.py

```diff
@@ -185,8 +185,10 @@ def __init__(self, table):
         takes an html table object (from lxml)
         """
         super(Table, self).__init__(table)
-        self.head = TableHead(table.find('thead'), parent=self) if table.find('thead') else None
-        self.body = TableBody(table.find('tbody') or table, parent=self)
+        table_head = table.find('thead')
+        self.head = TableHead(table_head, parent=self) if table_head is not None else None
```

Comment:
This statement would again be equivalent to doing the `if table_head` check which causes the error.

```
.../site-packages/tablepyxl/style.py:189: FutureWarning: The behavior of this method will change in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
  self.head = table_head and TableHead(table_head, parent=self)
```